pom.xml

```diff
@@ -143,6 +149,81 @@
 
   <build>
     <plugins>
+      <!--
+      Demo: Copy dependencies so runtime-agent.jar location is know for -javaagent and also helps with iq-cli scans.
+      -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>dependency-copy-dependencies</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/lib</outputDirectory>
+              <stripVersion>false</stripVersion>
+              <includeScope>compile</includeScope>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- Demo: CLM plugin -->
+      <plugin>
+        <groupId>com.sonatype.clm</groupId>
+        <artifactId>clm-maven-plugin</artifactId>
+        <version>${clm.maven.plugin.version}</version>
+        <configuration>
+          <applicationId>${artifactId}</applicationId>
+          <stage>stage-release</stage>
+          <skip>true</skip>
+        </configuration>
+        <executions>
+          <execution>
```

Comment:
2 executions are required:
- one before tests execution, to get scan results for vulns signature
- another after tests execution, just to get a policy update now that labels have been set at runtime

this can be done with something like
```
          <execution>
            <id>sonatype-prepare</id>
            <phase>test-compile</phase>
            <goals>
              <goal>evaluate</goal>
            </goals>
          </execution>
          <execution>
            <id>sonatype-reevaluate-after-run</id>
            <phase>prepare-package</phase>
            <goals>
              <goal>evaluate</goal>
            </goals>
          </execution>

```

notice that `index` is not useful when we use `evaluate`

Reply:
Thanks